src/main/java/com/bugsnag/HandledState.java

```diff
@@ -21,31 +25,53 @@ public String toString() {
     }
 
     private final SeverityReasonType severityReasonType;
+    private final Map<String, String> severityReasonAttributes;
```

Comment:
I added the attributes in here to minimize the amount of refactoring, though it has made this class a little clunkier.